charts/nextcloud/files/notify_push.sh.tpl

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+/var/www/html/occ app:install notify_push
```

Comment:
```suggestion
/var/www/html/occ app:enable notify_push
```
This will both install and enable the app. I'd expect that the app is automatically enabled, if it was disabled but installed previously.